doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,92 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
```

Comment:
Could we have it so that there is not multiple instances of the same function in the doc?
E.g., [this](https://xanaduai-pennylane--4250.com.readthedocs.build/en/4250/code/api/pennylane.FermiA.html) and [this](https://xanaduai-pennylane--4250.com.readthedocs.build/en/4250/code/api/pennylane.fermi.FermiA.html)?

To do this, we need to change here so that `qml.fermi.FermiA` is not included in the docs and we instead point explicitly to `qml.FermiA`.

Reply:
Yes, will address in a new PR.